remake.yml

```diff
@@ -15,21 +15,29 @@ targets:
   all:
     depends: 3_visualize/out/figure_1.png
 
-  site_data:
-    command: download_nwis_data()
-  
-  1_fetch/out/site_info.csv:
-    command: nwis_site_info(fileout = '1_fetch/out/site_info.csv', site_data)
+  1_fetch/out/site_data/01427207.csv:
+    command: download_nwis_site_data(target_name)
+  1_fetch/out/site_data/01432160.csv:
+    command: download_nwis_site_data(target_name)
+  1_fetch/out/site_data/01435000.csv:
+    command: download_nwis_site_data(target_name)
+  1_fetch/out/site_data/01436690.csv:
+    command: download_nwis_site_data(target_name)
+  1_fetch/out/site_data/01466500.csv:
+    command: download_nwis_site_data(target_name)
+    
+  all_site_files:
+    command: hash_dir_files(directory = I('1_fetch/out/site_data'), dummy = I('2020-07-01'))
+    depends:
+      - 1_fetch/out/site_data/01466500.csv
+      - 1_fetch/out/site_data/01436690.csv
+      - 1_fetch/out/site_data/01435000.csv
+      - 1_fetch/out/site_data/01432160.csv
+      - 1_fetch/out/site_data/01427207.csv
 
-  site_data_clean:
-    command: process_data(site_data)
-
-  site_data_annotated:
-    command: annotate_data(site_data_clean, site_filename = '1_fetch/out/site_info.csv')
-
-  site_data_styled:
-    command: style_data(site_data_annotated)	
+  site_data_processed:
+    command: process_data(all_site_files, site_filename = '1_fetch/out/site_info.csv')
```

Comment:
You deleted the '1_fetch/out/site_info.csv' target but are using it as an argument here...

Reply:
Oops, I must have done that while deleting the other unnecessary targets